### ISSUE TITLE
Execution Configuration Dialog

### ISSUE THREAD

**Issue by vasudev13** [scorelab/TensorMap#124]
To make things easier and avoid confusion, we can have selects for configuration settings. Moreover, while defining the Optimizer, there should be an number input for Learning Rate. 

Chip Multiselect was used for metrics as having multiple metrics is common while evaluating a network(though same can be achieved using checkboxes) . Plus, if an additional functionality is to have Custom Metrics (very common in ML Competitions), allowing user to define an custom python function in a textarea.

![Peek 2020-03-24 14-17](https://user-images.githubusercontent.com/30664477/77406277-67cce200-6dda-11ea-907a-0f2996fa1efb.gif)

PR will be established, if this fits well within the application's design and intuition.

**Comment by srivastava9** [comment]
@vasudev13 Do check the backend if it contain such options :)

**Comment by vasudev13** [comment]
As in keras's backend? or application's backend. I believe earlier options were hard-coded in config object, I have only changed the UI elements. I do agree a list can be populated from a API, but I don't think any such implementation is present as of now. I could be wrong.
